File: Pokua/Exercise9.java
import java.util.Scanner;

public class Exercise9 {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.println("Enter your weight in kg: ");
        double weight = input.nextDouble();

        System.out.println("Enter your height in m: ");
        double height = input.nextDouble();

        double bmi = weight/Math.pow(height, 2);
        
        if (bmi<18.5){
            System.out.println("Your BMI is "+bmi+", you are underweight");
        } else if (bmi<=24.9 && bmi>=18.5){
            System.out.println("Your BMI is "+bmi+", your weight is normal");
        }else if (bmi<=29.9 && bmi>=25){
            System.out.println("Your BMI is "+bmi+", you are overweight");
        }else if(bmi<=34.9 && bmi>=30){
            System.out.println("Your BMI is "+bmi+", you are obese");
        }else{
            System.out.println("Your BMI is "+bmi+", you are clinically obese");
        }

        input.close();
    }
    
}

File: Sunset/Basics/CODE_CHALLENGES/Day 3/exercise9.js
/*Write a program that interprets the Body Mass Index (BMI) on a user's weight and height.
 It should tell them the interpretation of thier BMI value.*/

 function calculateBMI(weight, height) {
    const bmi = Math.floor(weight / (height * height));
    return bmi;
  }
  
  /*const w = +prompt("Enter your weight in kg");
  const h = +prompt("Enter your heght in meters");*/
  
  let bmi = calculateBMI(100, 2.19);
  var observation;

  if (bmi < 18.5) {
    observation = 'Underweight';
  }else if(bmi < 24.9){
    observation = 'Normal weight';
  }else if(bmi < 29.9){
    observation = 'Overweight';
  }else if(bmi < 34.9) {
    observation = 'obese';
  }else{
    observation = 'clinically Obese'
  }

  console.log(`Your BMI is ${bmi}, you are ${observation}`)
  